app-shared-tests/src/main/java/com/appunite/loudius/AbsReviewersScreenTest.kt

```diff
@@ -111,6 +111,8 @@ abstract class AbsReviewersScreenTest {
     @Test
     fun whenClickOnNotifyAndDoNotCommentThenShowError() {
         with(integrationTestRule) {
+            Register.set404Error(mockWebServer)
```

Comment:
**suggestion**: the name doesn't tell when exactly the error is returned.

I guess adding just add the code would be the best:
```kotlin
mockWebServer.register {
            expectThat(it).url.and {
                get("host") { host }.isEqualTo("api.github.com")
                path.isEqualTo("/repos/exampleOwner/exampleRepo/issues/1/comments")
            }

            MockResponse().setResponseCode(404)
        }
```